tests/middlewareTypes.test.tsx

```diff
@@ -665,3 +557,41 @@ describe('more complex state spec with subscribeWithSelector', () => {
     TestComponent
   })
 })
+
+describe('create with explicitly annotated mutators', () => {
+  it('subscribeWithSelector & persist', () => {
+    const useStore = create<
+      CounterState,
+      [
+        ['zustand/subscribeWithSelector', never],
+        ['zustand/persist', Partial<CounterState>]
+      ]
+    >(
```

Comment:
Thanks. I'm sure some people prefer this style.

We might need more patterns of combinations.

Reply:
Yeah, I think we will need them eventually. But, it's fine for this PR.